src/dsp_tools/commands/excel2json/resources.py

```diff
@@ -154,7 +154,7 @@ def _row2resource(
 
     cards = []
     for j, detail_row in details_df.iterrows():
-        j = int(str(j))  # j is a label/index/hashable, but we need an int
+        j = int(str(j))  # noqa: PLW2901 (redefined-loop-name) (j is a label/index/hashable, but we need an int)
```

Comment:
could this be changed rather than ignored?

Reply:
maybe we could do this directly when we hand over the variable and don't assign it, that should work no?

Reply:
This is outdated, I found a better solution for it.